Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue3139.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 3139, "DisplayActionSheet is hiding behind Dialogs", PlatformAffected.UWP)]
+	public class Issue3139 : TestContentPage
+	{
+		protected override async void Init()
+		{
+			var alertTask = DisplayAlert("AlertDialog", "If this text is visible, then the test is failed", "Cancel");
```

Comment:
"If this text is visible, then the test is failed" is confusing - that text _will_ be visible when the user dismisses the ActionSheet. This needs to be reworded (or the test needs to be rearranged) to avoid this confusion.